tests/PHPStan/Rules/Methods/data/static-method-call-statement-no-side-effects-phpdoc.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace StaticMethodCallStatementNoSideEffects;
+
+class BzzStatic {
+	static function regular(string $a): string {return $a;}
+	/**
+	 * @phpstan-pure
+	 */
+	static function pure1(string $a): string {return $a;}
+	/**
+	 * @psalm-pure
+	 */
+	static function pure2(string $a): string {return $a;}
+}
+BzzStatic::regular('test');
+BzzStatic::pure1('test');
+BzzStatic::pure2('test');
```

Comment:
This should be put into an anonymous function so that it's not called with runtime reflection.